services/order.py

```diff
@@ -0,0 +1,38 @@
+from datetime import datetime
+
+from django.db import transaction
+from django.db.models import QuerySet
+
+from db.models import Order, Ticket, User
+
+
+def create_order(
+        tickets: list[dict],
+        username: str,
+        date: datetime = None
+) -> Order:
+    with transaction.atomic():
+        order = Order()
+        user = User.objects.get(username=username)
```

Comment:
Consider handling the `DoesNotExist` exception for the `User.objects.get(username=username)` call to manage cases where a user with the given username does not exist. This will prevent the application from crashing due to an unhandled exception.